pkg/objstore/azure/azure.go

```diff
@@ -0,0 +1,318 @@
+package azure
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+
+	blob "github.com/Azure/azure-storage-blob-go/2018-03-28/azblob"
+	"github.com/improbable-eng/thanos/pkg/objstore"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+	yaml "gopkg.in/yaml.v2"
+)
+
+const (
+	opObjectsList  = "ListBucket"
+	opObjectInsert = "PutObject"
+	opObjectGet    = "GetObject"
+	opObjectHead   = "HEADObject"
```

Comment:
Not your fault. But it is not consistent capitalisation for `Head` vs other constants.

Reply:
I am not sure I understand what should I do with this one. :(

Reply:
I think @jojohappy meant that you should be consistent in style. So either do `GETObject` and rest or change `HEADObject` to `HeadObject` (: